app/models/cdr_export.rb

```diff
@@ -0,0 +1,52 @@
+# == Schema Information
+#
+# Table name: cdr_exports
+#
+#  id           :integer          not null, primary key
+#  status       :string           not null
+#  fields       :string           default([]), is an Array
+#  filters      :json             not null
+#  callback_url :string
+#  created_at   :datetime
+#  updated_at   :datetime
+#
+
+class CdrExport < Yeti::ActiveRecord
+  self.table_name = 'cdr_exports'
+
+  STATUS_PENDING = 'Pending'.freeze
+  STATUS_COMPLETED = 'Completed'.freeze
+  STATUS_FAILED = 'Failed'.freeze
+  ALLOWED_FILTERS = [
+    #todo: add possible filters
+    'time_start_less_or_eq',
+    'time_start_greater_or_eq',
+    'success_eq'
+  ].freeze
+  #any cdr columns
+  ALLOWED_FIELDS = Cdr::Cdr.column_names.freeze
+
+  validates_presence_of :status, :fields, :filters
+  validate do
+    # todo add requires at time_start filter
+  end
+  validate do
+    extra_fields = fields - ALLOWED_FIELDS
+    if extra_fields.any?
```

Comment:
Imo there should be 1 abstract Export Class with minimal validation logic, to extend it and reuse with different logic

Reply:
why we need this overhead at this level?

Reply:
@gigorok 
>> to extend it and reuse with different logic